### PR TITLE
Standardize logging

### PR DESCRIPTION
**Pull request by labrys**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/search/backlog.py [hunk 1]
```diff
@@ -102,10 +90,10 @@ def search_backlog(self, which_shows=None):
         from_date = datetime.date.fromordinal(1)
 
         if not which_shows and self.forced:
-            logger.log(u'Running limited backlog search on missed episodes from last {0} days'.format(app.BACKLOG_DAYS))
+            log.info(u'Running limited backlog search on missed episodes from last {0} days'.format(app.BACKLOG_DAYS))
```
Comment (p0psicles) [inline medusa/search/backlog.py]: Change to lazy log.

File: medusa/search/backlog.py [hunk 2]
```diff
@@ -116,13 +104,13 @@ def search_backlog(self, which_shows=None):
...
             for season, segment in iteritems(segments):
-                self.currentSearchInfo = {'title': cur_show.name + " Season " + str(season)}
+                self.currentSearchInfo = {'title': cur_show.name + ' Season ' + str(season)}
 
                 backlog_queue_item = BacklogQueueItem(cur_show, segment)
                 app.search_queue_scheduler.action.add_item(backlog_queue_item)  # @UndefinedVariable
 
             if not segments:
-                logger.log(u"Nothing needs to be downloaded for %s, skipping" % cur_show.name, logger.DEBUG)
+                log.debug(u'Nothing needs to be downloaded for {0}, skipping', cur_show.name)
```
Comment (p0psicles) [inline medusa/search/backlog.py]: We can do this? Positional using {0}?. personal I would then use default logging here. Like %,..

Comment (labrys) [inline medusa/search/backlog.py]: That's what `{0}` is, positional formatting for `.format`.  The advantage to using `.format` is its a lot more flexible.

File: medusa/search/backlog.py (hunk 1 above)
Comment (labrys) [inline medusa/search/backlog.py]: Good catch :)

File: medusa/search/backlog.py (hunk 2 above)
Comment (fernandog) [inline medusa/search/backlog.py]: Nothing needs to be downloaded for show '{show}'

Comment (labrys) [inline medusa/search/backlog.py]: @fernandog I dont think adding show is necessary.  If you would like the quotes around it I'm fine with that

`Nothing needs to be downloaded for "Game of Thrones"`

Comment (fernandog) [inline medusa/search/backlog.py]: ok. let's add single quotes then

**Pull request merged**